konnectivity-client/pkg/client/client.go

```diff
@@ -115,6 +115,12 @@ func (t *grpcTunnel) serve(c clientConn) {
 					connid: resp.ConnectID,
 				}
 			}
+
+			if resp.Error != "" {
+				// On dial error, avoid leaking serve goroutine.
```

Comment:
Can we log the error?

Reply:
Not against this. Worth noting that I believe the resp.Error will be passed back to Dial() through the channel and then returned up to the caller of Dial. I'm assuming that caller will log it with more context.

Reply:
Good point. I missed line 114. Then the log is less important.